src/app/shared/components/number-plate/number-plate.component.html

```diff
@@ -1,3 +1,3 @@
-<span class="govuk-number-plate">
-  <ng-content></ng-content>
+<span class="govuk-number-plate" [class.secondary]="isSecondary">
```

Comment:
I think `ngClass` might be the preferred way of doing this
```suggestion
<span class="govuk-number-plate" [ngClass]="{'secondary': isSecondary}">
```